src/const_eval.md

```diff
@@ -42,7 +42,12 @@ r[const-eval.const-expr.path-item]
   Recursively defining constants is not allowed.
 
 r[const-eval.const-expr.path-static]
-* Paths to [statics]. These are only allowed within the initializer of a static.
+* Paths to [statics] with these restrictions and observations.
```

Comment:
```suggestion
* Paths to [statics] with these restrictions:
```
Sorry, just noticed this now. What are the "observations"?

Reply:
I had a `:` in my suggestion here, which you turned back into a `.` for some reason -- seems odd to introduce a list with a `.` like this. It's not a big deal, I just wonder why you didn't simply click the button to apply my suggestion.

Reply:
Oh sorry. I did not notice that there was a change suggestion when reading the email. I would have just switch to colon.